minvws/nl-kat-coordination
# Feature/upload multiple files at once to bytes

### Changes

This makes the raw endpoint accept multiple files by default. Note that I think that utilizing the new functionality belongs in a subsequent PR, so there is no functionality added here and everything is backward compatible at the BytesClient level at least, mostly to keep PR's small and focused.

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3113

### Demo

-

### QA notes

These changes do not add functionality but affect Bytes, Boefjes and Rocky. Please verify that the boefjes still run properly, we can download raw files from Rocky, and we can still upload raw files from Rocky!

---

### Code Checklist


---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
